### DISCUSSION TITLE
Console login as non-admin?

### DISCUSSION THREAD

**Discussion by dgchrt** [hasura/graphql-engine#7376] [Question]
Is it possible to enable console login for non-admin users, so that they could use the GraphiQL tool?

**Comment by reinoldus** [comment]
This seems to be possible only with hasura cloud or the self-hosted pro version:
![image](https://user-images.githubusercontent.com/2091290/128838261-9c664f0a-6204-468a-a4c4-f4d19c81172b.png)



You can work around it by deploying a GraphiQL yourself and hide it behind a .htaccess

**Comment by praveenweb** [comment]
We have also built an online version of GraphiQL that you can use for just testing against any GraphQL endpoint.

It is available at [https://graphiql-online.com](https://graphiql-online.com). You can add your custom headers etc.

**Reply by deathemperor** [reply]
been looking for a tool like this for a long time since I've started using Hasura! This is fantastic for non admin users to write queries.
